app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -65,24 +70,32 @@ export class ArticleCoverImage extends Component {
           />
         )}
         <div className="flex items-center">
-          <Button variant="outlined" className="mr-2 whitespace-nowrap">
-            <label htmlFor="cover-image-input">{uploadLabel}</label>
-            <input
-              id="cover-image-input"
-              type="file"
-              onChange={this.handleMainImageUpload}
-              accept="image/*"
-              className="w-100 h-100 absolute left-0 right-0 top-0 bottom-0 overflow-hidden opacity-0 cursor-pointer"
-              data-max-file-size-mb="25"
-            />
-          </Button>
-          {mainImage && (
-            <Button
-              variant="ghost-danger"
-              onClick={this.triggerMainImageRemoval}
-            >
-              Remove
-            </Button>
+          {uploadingImage ? (
+            <span class="lh-base pl-1 border-0 py-2 inline-block">
+              <Spinner /> Uploading...
+            </span>
+          ) : (
+            <Fragment>
```

Comment:
Since we are using Preact X now, [fragments](https://preactjs.com/guide/v10/components/#fragments) can be used now. The terser syntax is preferred, e.g. `<><SomeComponent</>` vs. `<Fragment><SomeComponent</Fragment>`.

I'll enable this in another PR as it requires some babel configuration changes.